Makefile

```
@@ -1,13 +1,25 @@
 TEST_TARGET := ./tests/
+PY_PACKAGE := ansible-bender
+KNIFE := knife
+
+build-knife:
+	ansible-bender build ./knife.yml registry.fedoraproject.org/fedora:29 $(KNIFE)
 
 check:
 	PYTHONPATH=$(CURDIR) pytest-3 -v $(TEST_TARGET)
 
 check-pypi-packaging:
-	podman run --rm -ti -v $(CURDIR):/src -w /src registry.fedoraproject.org/fedora:29 bash -c '\
-		pip3 install . \
-		&& ab --help \
-		&& ab build --help'
+	podman run --rm -ti -v $(CURDIR):/src -w /src $(KNIFE) bash -c '\
+		set -x \
+		&& pip3 install . \
+		&& ansible-bender --help \
+		&& ansible-bender build --help \
+		&& pip3 show $(PY_PACKAGE) \
+		&& python3 ./setup.py sdist \
```

You should also always publish a binary wheel along with a source distribution.
```suggestion
		&& python3 ./setup.py sdist bdist_wheel \
```

---

Yes, this is just for sake of checking, good suggestion though.